mappings/net/minecraft/util/math/BoundingBox.mapping

```diff
@@ -31,27 +32,65 @@ CLASS none/bcs net/minecraft/util/math/BoundingBox
 		ARG 3 maxX
 		ARG 4 maxY
 		ARG 5 maxZ
+	METHOD a intersectX (DLnone/bcv;Lnone/bcv;)Lnone/bcv;
+		ARG 0 x
+		ARG 1 begin
+		ARG 2 end
+	METHOD a union (Lnone/bcs;)Lnone/bcs;
+		ARG 0 other
+	METHOD a xOffsetForCollision (Lnone/bcs;D)D
+		ARG 0 other
+		ARG 1 xOffset
+	METHOD a offset (Lnone/bcv;)Lnone/bcs;
+		ARG 0 offset
 	METHOD a contains (Lnone/bcv;Lnone/bcv;)Z
 		ARG 0 a
 		ARG 1 b
+	METHOD a nearest (Lnone/bcv;Lnone/bcv;Lnone/bcv;)Z
+		ARG 0 reference
+		ARG 1 falseIfNearest
+		ARG 2 trueIfNearest
```

Comment:
This is an awful function. I should probably have left it without names.
When `ARG 1` is the nearest point to `reference`, the function returns `false`.
When `ARG 2` is the nearest point to `reference`, the function returns `true`.